modules/platform/login/nuxeo-platform-login-openid/src/main/java/org/nuxeo/ecm/platform/oauth2/openid/OpenIDConnectProviderDescriptor.java

```diff
@@ -161,6 +168,10 @@ public String getIcon() {
 
     @Override
     public boolean isEnabled() {
+        if (isBlank(clientId) || isBlank(clientSecret)) {
+            log.info("OpenId provider: {} is disabled because clientId and/or clientSecret are empty", name);
```

Comment:
Having a log in the getter means that this is going to potentially get logged numerous times, no?

Reply:
It should be done once per Component#start calls. Otherwise, I can remove it, not sure that an info log is going to be useful at some point, wdyt?

Reply:
I don't think it's the descriptor's role to warn of such things. For me the descriptor is a pure carrier of data without any logic (and in general even the defaults should probably be elsewhere, in a service).